en/views/cells.rst

```diff
@@ -276,3 +276,28 @@ This will allow us to define the option when creating the cell::
 Cell options are handy when you want data available as properties allowing you
 to override default values.
 
+Using Helpers inside a Cell
+===========================
+
+Since cells have their own View scope they don't share the helpers defined in
+`src/View/AppView.php`. You need to load those helpers again inside your cell::
```

Comment:
I think we use terms like "in your View" since it's not necessarily src/View/AppView.

Reply:
Defined in e.g. .. 
Might also work